docs/source/services/timelock_service/reverse-migration.rst

```diff
@@ -0,0 +1,140 @@
+.. _timelock-reverse-migration:
+
+Reverse Migration
+=================
+
+.. danger::
+
+   AtlasDB does not natively support reverse migrations. Improperly executing reverse migration from external timestamp
+   and lock services can result in **SEVERE DATA CORRUPTION**! Please contact the AtlasDB team before attempting a
+   reverse migration.
+
+If one wishes to downgrade from an external TimeLock Server to embedded timestamp and lock services, one can perform
```

Comment:
I find this a bit of a mouthful. How about:

"While the migration from an embedded to external timelock service is automatic, the reverse is a manual process. The steps are the inverse of <Manual Migration to Timelock Server>, with an additional step to update the timestamp bound of the embedded service."